pkg/v1/providers/ytt/03_customizations/capabilities/add_capabilities.yaml

```diff
@@ -60,7 +60,6 @@ spec:
                             group:
                               description: Group is the API group to check for in the
                                 cluster.
-                              minLength: 1
```

Comment:
Is this correct? I think required and minLength validate 2 different things. This YAML used to say that the group field MUST be present and it MUST contain at least 1 character.

I just want to make sure we intend to say "group is no longer required and if it is present it can be empty", compared to "group is no longer required, but if you specify one it still has a min length"

Reply:
Or I guess there's a 3rd option, "group is a required field but it can be empty". This seems closest to what the original issue asked for.

Reply:
I played around with `minLength` and `omitempty` and I still think the current code makes sense.

If we enforce `minLength=1`, it doesn't allow authors to explicitly set `group: ""` to refer to k8s core API group and they have to rely on omitting the field itself as way to set use core group. I think that's confusing; I'd expect both the absence of the field and setting it to empty string to be the same.

With the current code, the behavior is pretty straightforward:

1. If `group` field is not present or is set to `""`, it refers to core group.
2. If `group` field is present and is set to ay value, it refers to a different API group.

I think length validation is not necessary at all. 

Let me know what you think!